lib/IRGen/IRGenModule.cpp

```diff
@@ -693,6 +693,7 @@ void IRGenModule::emitAutolinkInfo() {
     }
     break;
   }
+  case llvm::Triple::COFF:  // for cygwin
```

Comment:
The autolinking section is used by the `swift-autolink-extract` to enable autolinking of the dependent libraries.